solutions/level1/printer.rb

```diff
@@ -1,7 +1,16 @@
 require_relative 'json_reader_writer'
 
 class Printer
-  def self.export_receipt(json)
+  def self.export_receipt(books, receipt)
+    json = merge_receipt_with_books(books, receipt)
     JsonReaderWriter.export_to_file(json)
   end
+
+  def self.merge_receipt_with_books(books, receipt)
+    receipt[:books] = []
+    books.each do |book|
```

Comment:
this can be written in a much shorter form like `receipt[:books] = books.map(&:to_h)`